docs/src/getting-started/installation.md

```diff
@@ -90,14 +80,65 @@ As for now, Starknet Foundry on Windows needs manual installation, but necessary
    $ sncast --version
    ```
 
-### Universal-Sierra-Compiler update
+## Universal-Sierra-Compiler update
 
 If you would like to bump the USC manually (e.g. when the new Sierra version is released) you can do it by running:
 
 ```shell
 $ curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh
 ```
 
+## Common Errors
+
+### No Version Set
+
+Users may encounter this error when trying to use `snforge` or `sncast` without setting a version:
+
+```shell
+No version is set for command snforge
+Consider adding one of the following versions in your config file at $HOME/.tool_versions
+starknet-foundry 0.32.0
+```
+
+This error indicates that `Starknet Foundry` version is unset. To resolve it, set the version globally using asdf:
+
+```shell
+$ asdf global starknet-foundry <version>
+```
+
+For additional information on asdf version management, see
+the [asdf](https://asdf-vm.com/guide/getting-started.html#_6-set-a-version)
+
+### Invalid Rust Version
+
+When running any `snforge` command, error similar to this is displayed
+
+```shell
+Compiling snforge_scarb_plugin v0.34.0
+error: package `snforge_scarb_plugin v0.34.0 cannot be built because it requires rustc 1.80.1 or newer, while the currently active rustc version is 1.76.0
```

Comment:
Looks like typo 😅 
```suggestion
error: package snforge_scarb_plugin v0.34.0 cannot be built because it requires rustc 1.80.1 or newer, while the currently active rustc version is 1.76.0
```